resources/lib/syncEpisodes.py

```diff
@@ -578,10 +575,10 @@ def __compareShows(self, shows_col1, shows_col2, rating=False, restrict=False):
 
                 if show_col2:
                     show = {'title': show_col1['title'], 'ids': {}, 'year': show_col1['year']}
-                    if 'tvdb' in show_col1['ids']:
-                        show['ids'] = {'tvdb': show_col1['ids']['tvdb']}
-                    if 'imdb' in show_col2 and show_col2['imdb']:
-                        show['ids']['imdb'] = show_col2['imdb']
+                    if show_col1['ids']:
+                        show['ids'].update(show_col1['ids'])
+                    if show_col2['ids']:
+                        show['ids'].update(show_col2['ids'])
```

Comment:
Same here, wouldn't this overwrite whatever the first one had set?

Reply:
Btw scrapers with multiple ids have landed, at least for krypton and above and for movies.
Will have to look into it, what we need to do to actually use them.

Reply:
Btw for reference this was the initial PR https://github.com/xbmc/xbmc/pull/10000

So we just need to add uniqueid handling for everything but episodes ;)